cmd/snapctl/task.go

```diff
@@ -100,14 +100,225 @@ func createTask(ctx *cli.Context) error {
 	return err
 }
 
+func stringValToInt(val string) (int, error) {
+	// parse the input (string) value as an integer (log a Fatal
```

Comment:
Maybe just word this comment as return an error instead of logging a fatal error message since we don't actually log a message in snapctl, but just return an error to the user.